### ISSUE TITLE
AI3 enhancement

### ISSUE THREAD

**Issue by StevenClontz** [TeamBasedInquiryLearning/library#29]
Right now it's always the best choice to integrate with respect to x. We should probably randomly invert the x and y values so more thought is required.

**Comment by TienChih** [comment]
I updated the generator and template to something Steven and I discussed in the past, something like https://tienchih.github.io/oxcalcbank/#/bank/AP3/1/.  Alas, I can't remember how to actually build the checkit from github.

**Comment by siwelwerd** [comment]
>Alas, I can't remember how to actually build the checkit from github.

You'll need to open a Codespace, but then see our Wiki (particularly [here](https://github.com/TeamBasedInquiryLearning/library/wiki/CheckIt-Authoring#workflow-to-author-checkit-in-our-codespace)) for the script to run to generate a Preview of an outcome

**Comment by siwelwerd** [comment]
I thought you were talking about building a preview to check before it is merged.

Once a PR is merged, the workflow will do a full build and also fully rebuild the Checkit banks.  So no need to worry about that, assuming the previews looked fine when we are deciding to approve a PR.

**Comment by TienChih** [comment]
O good thats great!  Some of these are basically C&P from other banks I wrote, so they should be fine.

**Comment by siwelwerd** [comment]
Yeah, then I'd suggest just making a different PR for each outcome, and link it to the relevant issue.  If you type something like "Closes #NNN" in the PR, where NNN is the number of the issue, merging will automatically close that issue, and there will be links going both ways so someone looking at this issue later can see what PR closed it if needed to track something down.

**Comment by TienChih** [comment]
https://github.com/TeamBasedInquiryLearning/library/pull/448

**Comment by cg2wilson** [comment]
#448 was merged!
